compose/docker-compose.yml

```diff
@@ -153,19 +166,72 @@ services:
       - ../src/python:/home/broker/python
     working_dir: "/home/broker/python"
     entrypoint: [ "python", "-m", "delivery.UbidotsWriter" ]
+    ports:
+      - "8006:8006" # Expose the metrics port
 
   frred:
     image: broker/python-base
     restart: unless-stopped
     env_file:
       - .env
+    environment:
+      - PYTHONPATH=/home/broker/broker/src/python:/home/broker/broker/test/python
     depends_on:
-      db:
-        condition: "service_healthy"
-      mq:
-        condition: "service_healthy"
+      - "db"
+      - "mq"
     volumes:
       - ../src/python:/home/broker/python
       - ${DATABOLT_SHARED_DIR}/raw_data:/raw_data
     working_dir: "/home/broker/python"
     entrypoint: [ "python", "-m", "delivery.FRRED" ]
+    ports:
+      - "8007:8007" # Expose the metrics port
+
+  x:
+    image: broker/python-base
+    restart: "no"
+    env_file:
+      - .env
+    environment:
+      - PYTHONPATH=/home/broker/broker/src/python:/home/broker/broker/test/python
+    depends_on:
+      - "db"
+      - "mq"
+    volumes:
+      - ..:/home/broker/broker
+    working_dir: "/home/broker/broker"
+    entrypoint: [ "./forever.sh" ]
+
+  prometheus:
+    image: prom/prometheus
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+    ports:
+      - "9090:9090"
```

Comment:
Should this port only be exposed in `prod.yml` and should it probably only be exposed locally, i.e., `127.0.0.1:9090:9090`

Reply:
Actually, does this need to be exposed at all or do we leave it only routed internally for grafana?

Reply:
Both prom/graf need to be exposed